migration/template/apply/00-template.sh

```diff
@@ -44,13 +44,29 @@ source "${ROOT}/scripts/migration/lib.sh"
 #     - helmfile_upgrade <sc|wc> <selectors...>
 
 run() {
+  local cluster="${2:-both}"
```

Comment:
Use the environment variable instead of an argument if you go with an environment variable.